src/main/java/com/jmb/springfactory/model/entity/WorkLog.java

```diff
@@ -0,0 +1,21 @@
+package com.jmb.springfactory.model.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToOne;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+public class WorkLog extends BaseEntity {
+
+  private Double hoursWorked;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "hoursWorked" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)